Project.toml

```diff
@@ -3,14 +3,14 @@ uuid = "5c1f47dc-42dd-5697-8aaa-4d102d140ba9"
 version = "0.2.1"
 
 [deps]
-IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
+IntervalArithmetic = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
 julia = "≥ 1.0.0"
-IntervalSets = "≥ 0.3.1"
+IntervalArithmetic = "≥ 0.16.0"
```

Comment:
Do we need this version? `IntervalArithmetic` is not available in julia v1.0.

Reply:
It is the IntervalArithmetic version that i'm using in v1.1; it has several features and bug fixes see https://github.com/JuliaIntervals/IntervalArithmetic.jl/releases/tag/v0.16.0 . But i can try with an older version.

Reply:
Good. In v1.1 we still use the newest version.

Reply:
Okay, used v0.14 and tests are passing now.